changelog.d/7009.bugfix

```diff
@@ -0,0 +1 @@
+Set `Referrer-Policy` header to `no-referrer` on media downloads.
```

Comment:
This is probably more a feature than a bugfix?

Reply:
I considered this PR related to #2150, which is why I marked it as bugfix. Happy to rename it to a feature though.

Reply:
Updated as a feature, but the branch name is still `bugfix-` and needs a new PR if we want to change that.

Reply:
The branch name should be fine. 👍 Will wait for @erikjohnston's opinion on whether it is a bugfix or feature.

Reply:
@erikjohnston ping

Reply:
Sorry, missed this. Thanks. Branch name doesn't matter